tests/integration/test_allowed_recipients_happy_path.py

```diff
@@ -92,12 +109,14 @@ def full_setup(
     lido_contracts,
     allowed_recipients_default_params,
     deployer,
+    ldo,
+    usdc
 ):
     deploy_tx = allowed_recipients_builder.deployFullSetup(
         trusted_caller,
-        lido_contracts.ldo,
         allowed_recipients_default_params.limit,
         allowed_recipients_default_params.period_duration_months,
+        [ldo, usdc],
```

Comment:
It's better to use tokens we actually will use together in the same tokens registry